README.md

```diff
@@ -1,12 +1,94 @@
-# EduHelp
+<h1 align="center"> EduHelp</h1>
+<p align="center">
+	<img src="https://user-images.githubusercontent.com/68437435/111764796-a00de000-88c9-11eb-88ba-56369207aabd.gif" width=300 height=300 alt="Banner">
+</p>
+
+[![Number of Contributors](https://img.shields.io/github/contributors/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/graphs/contributors)
+  [![Issues opened](https://img.shields.io/github/issues/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp)
+  [![Issues closed](https://img.shields.io/github/issues-closed/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/issues)
+  [![PRs open](https://img.shields.io/github/issues-pr/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/pulls)
+  [![PRs closed](https://img.shields.io/github/issues-pr-closed/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/pulls)
+  ![Repo size](https://img.shields.io/github/repo-size/CrewMates-Open-Source/EduHelp)
+  [![Open Source Love svg2](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
+ 
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com)
+
 ## What It Does ? :thinking:
```

Comment:
Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.